crates/holochain/src/conductor/cell.rs

```diff
@@ -314,23 +313,7 @@ impl Cell {
                         expires_at,
                     };
 
-                    tasks.push(
-                        self.call_zome(
-                            match ZomeCall::try_from_params(
-                                self.conductor_handle.keystore(),
-                                zome_call_params,
-                            )
-                                .await
-                            {
-                                Ok(zome_call) => zome_call,
-                                Err(e) => {
-                                    error!("scheduled zome call error in try_from_unsigned_zome_call: {:?}", e);
-                                    continue;
-                                }
-                            },
-                            None,
-                        ),
-                    );
+                    tasks.push(self.call_zome(zome_call_params, None));
```

Comment:
Scheduled tasks no longer require authentication as they're internal to the conductor.